tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/RecoveryResource.java

```diff
@@ -115,13 +115,13 @@ public Response phase2(@HeaderParam(LRA_HEADER) URI lraId) {
 
         // assert compensate has been called
         int compensations = lraMetricService.getMetric(LRAMetricType.Compensated, lraId, RecoveryResource.class.getName());
-        if (compensations != 1) {
+        if (compensations < 1) {
             return assertFailedResponse("Compensate on restarted service should have been called. Was " + compensations);
         }
 
         // assert after LRA has been called
         int afterLRACalls = lraMetricService.getMetric(LRAMetricType.Cancelled, lraId, RecoveryResource.class.getName());
-        if (afterLRACalls != 1) {
+        if (afterLRACalls < 1) {
```

Comment:
afterLRACalls == 0 is more efficient (and readable)

Reply:
same comment as above